services/handlers/authentication.ts

```diff
@@ -2,7 +2,10 @@ import logger from '../utilities/logger'
 import { Pool } from 'pg'
 import { get, isEmpty } from 'lodash'
 import { hash, compare } from 'bcrypt'
-import { sign } from 'jsonwebtoken'
+import { sign, verify } from 'jsonwebtoken'
+
+const secret = process.env.JWT_SECRET
+const algorithm = 'HS512'
```

Comment:
Don't worry about hard-coding it, we won't be able to change our encoding until accessible quantum computing is widely available :)

Reply:
I'd rather write algorithm in HS512 to be honest

Reply:
@shakedlokits

Reply:
Fair enough 👍